src/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -4327,6 +4327,11 @@ void MethodContext::dmpCanInlineTypeCheckWithObjectVTable(DWORDLONG key, DWORD v
 }
 BOOL MethodContext::repCanInlineTypeCheckWithObjectVTable(CORINFO_CLASS_HANDLE cls)
 {
+    AssertCodeMsg(CanInlineTypeCheckWithObjectVTable != nullptr, EXCEPTIONCODE_MC, "No map for CanInlineTypeCheckWithObjectVTable");
+    if (CanInlineTypeCheckWithObjectVTable == nullptr)
```

Comment:
I think this part is not necessary. `AssertCodeMsg(CanInlineTypeCheckWithObjectVTable != nullptr, EXCEPTIONCODE_MC, "No map for CanInlineTypeCheckWithObjectVTable");` is enough.

Reply:
Yes, thanks - makes sense, and I verified that it simply fails the current MC without aborting.